mtags/src/main/scala-3/scala/meta/internal/pc/completions/Completions.scala

```diff
@@ -556,10 +561,9 @@ class Completions(
       ) // !sym.info.typeSymbol.is(Flags.Method) does not detect Java methods
 
     private def isNotClassOrTraitOrTheyAreValidForPos(sym: Symbol) =
-      cursorPositionCondition.isClassOrTraitValidForPos || sym.is(
-        Flags.Method
-      ) || (!sym.isClass && !sym.info.typeSymbol
-        .is(Trait))
+      cursorPositionCondition.isClassOrTraitValidForPos ||
```

Comment:
```suggestion
      cursorPositionCondition.isClassTraitOrPackageValidForPos ||
```

Reply:
Maybe `isNotValidObject` ?

Reply:
`isObjectInvalidForPos`

Reply:
but that would be cryptic a little bit for later maintenance

Reply:
But long names are difficult to differentiate when looking at the code.

Reply:
then, perhaps the longer explanation can go in scaladoc

Reply:
Ok moved it out of the method, so I will leave it as is.